### DISCUSSION TITLE
Allow for aligned search of Float and Double

### DISCUSSION THREAD

**Discussion by suXinjke** [RetroAchievements/RAIntegration#1125] [Completed]
It's quite rare for floating point values not to be stored properly aligned, so searching for float value changes results in lot of garbage results. I wish we could search for those values same way we can do 32-bit (aligned) search.

**Comment by redwizard42** [comment]
This was on my short list of stuff to look into doing, but got distracted by working on sets. I'll see about making at attempt at it.
